packages/strapi/lib/reload.js

```diff
@@ -20,7 +20,7 @@
 module.exports = function reload() {
   try {
     const reload = function() {
-      if (cluster.isWorker && process.env.NODE_ENV === 'development' && this.config.environments[this.config.environment].server.reload === true) process.send('message');
+      if (cluster.isWorker && process.env.NODE_ENV === 'development' && this.config.environments[this.config.environment].server.reload === true) process.send('reload');
```

Comment:
Again on this one. You should test that `this.config.environments[this.config.environment].server` is an object or use Lodash `get()` method instead.